lib/shoryuken/manager.rb

```diff
@@ -3,7 +3,7 @@ class Manager
     include Util
 
     BATCH_LIMIT        = 10
-    HEARTBEAT_INTERVAL = 0.1
+    HEARTBEAT_INTERVAL = 1
```

Comment:
@waynerobinson I'm testing `HEARTBEAT_INTERVAL = 1`, as I'm also calling `dispatch` in the `processor_done` callback.